docs/template-helpRequest.md

```diff
@@ -0,0 +1,66 @@
+**Help Request**
+
+**Issue Title:** Help Request: [summary of your problem]
+
+
+**Operating System**
+
+
+**JDK Version**
+
+
+**Eclipse Version**
+
+
+**Firefox / Chrome Version (where applicable)**
```

Comment:
Since the above four information are all very short, consider making them into bullet points instead, something like:
- **A:**
- **B:**
- **C:**
- **D:**